src/sentry/static/sentry/app/views/performance/traceDetails/content.tsx

```diff
@@ -588,6 +590,15 @@ class TraceDetailsContent extends React.Component<Props, State> {
               <FeatureBadge type="beta" />
             </Layout.Title>
           </Layout.HeaderContent>
+          <Layout.HeaderActions>
+            <ButtonBar gap={1}>
+              <DiscoverButton
+                to={traceEventView.getResultsViewUrlTarget(organization.slug)}
```

Comment:
Should drop duration(and depending on how the url is constructed any performance specific field) from this, since snuba will go to the txn table if duration is a column and none of the errors for the trace will show up